src/translations/en.json

```diff
@@ -1480,7 +1480,9 @@
             "area": "Area",
             "integration": "Integration",
             "battery": "Battery"
-          }
+          },
+          "delete": "DELETE",
```

Comment:
Keeps this lowercase, the styling will make it uppercase

Reply:
Fixed.
I also changed some other button labels to lowercase.